### ISSUE TITLE
OSError: [Errno 113] No route to host

### ISSUE THREAD

**Issue by mkzia** [jupyter/nbgrader#1213]
### Operating system
Ubuntu 18.04

### `nbgrader --version`
Latest master branch and 0.60

### `jupyterhub --version` (if used with JupyterHub)
1.0.0
### `jupyter notebook --version` 
6.0.1

### Expected behavior
Course tab should list course

### Actual behavior
Seeing this error message
```Traceback (most recent call last): File "/srv/nbgrader/nbgrader/nbgrader/server_extensions/course_list/handlers.py", line 175, in get local_courses = yield self.check_for_local_formgrader(config) File "/usr/local/lib/python3.6/dist-packages/tornado/gen.py", line 735, in run value = future.result() File "/usr/local/lib/python3.6/dist-packages/tornado/gen.py", line 742, in run yielded = self.gen.throw(*exc_info) # type: ignore File "/srv/nbgrader/nbgrader/nbgrader/server_extensions/course_list/handlers.py", line 69, in check_for_local_formgrader response = yield http_client.fetch(url, headers=header) File "/usr/local/lib/python3.6/dist-packages/tornado/gen.py", line 735, in run value = future.result() OSError: [Errno 113] No route to host```

### Steps to reproduce the behavior
Install by doing `bash restart_demo.sh demo_one_class_multiple_graders`

This problem is flaky. I spun up a new instance and now I am getting this problem. I wasn't getting this problem before. 

When I first saw this error message, I downgraded Tornado to 5.1.1 and that solved the problem. But now this problem is happening again and downgrading to Tornado 5.1.1 does not help.

**Comment by mkzia** [comment]
Actually, the tornado related error is
```    Traceback (most recent call last):
      File "/usr/local/lib/python3.6/dist-packages/tornado/web.py", line 1699, in _execute
        result = await result
      File "/usr/local/lib/python3.6/dist-packages/oauthenticator/oauth2.py", line 207, in get
        user = await self.login_user()
      File "/usr/local/lib/python3.6/dist-packages/jupyterhub/handlers/base.py", line 655, in login_user
        authenticated = await self.authenticate(data)
      File "/usr/local/lib/python3.6/dist-packages/jupyterhub/auth.py", line 383, in get_authenticated_user
        authenticated = await maybe_future(self.authenticate(handler, data))
      File "/usr/local/lib/python3.6/dist-packages/oauthenticator/github.py", line 115, in authenticate
        resp = await http_client.fetch(req)
    tornado.simple_httpclient.HTTPStreamClosedError: Stream closed```
This is solved by downgrading tornado to 5.1.1 as noted here https://github.com/jupyterhub/jupyterhub/issues/2448. 

Not sure what the cause of my error is.

**Comment by mkzia** [comment]
Turns out I had to flush my DNS cache using  `systemd-resolve --flush-caches`
